packages/react/src/components/Page/__snapshots__/EditPage.story.storyshot

```diff
@@ -159,25 +162,26 @@ exports[`Storybook Snapshot tests and console checks Storyshots 1 - Watson IoT/
   >
     <div
       className="EditPage__StyledEditPage-gpmt4s-0 gHSBnh"
+      data-testid="edit-page"
     >
       <div
         className="Hero__StyledHero-nej8yx-0 fTphDS"
-        data-testid="hero"
+        data-testid="edit-page-hero"
       >
         <div
           className="Hero__StyledTitleSection-nej8yx-1 imCjnH"
         >
           <div
             className="Hero__StyledTitle-nej8yx-2 bbRUKN"
-            data-testid="hero-title"
+            data-testid="edit-page-hero-title"
           >
             Page Title
           </div>
           <button
             aria-label="Close,"
             className="bx--modal-close"
             data-modal-close={true}
-            data-testid="hero-close-button"
```

Comment:
These changes around `data-testid="hero"`, data-testid="hero-title"`, data-testid="hero-close-button", and `data-testid="hero-description" look like breaking changes, but these testIds were just added in an earlier testId PR. https://github.com/carbon-design-system/carbon-addons-iot-react/blame/e643b3468bc4fa30b0d635e5c4de9df573aa23aa/packages/react/src/components/Page/__snapshots__/EditPage.story.storyshot#L170